### PR TITLE
grim: init at 1.0

### PR DESCRIPTION
**Pull request by buffet**
###### Motivation for this change
Screenshots are neat.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/graphics/grim/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    rev = "v${version}";
+    sha256 = "1mpmxkzssgzqh9z263y8vk40dayw32kah66sb8ja7yw22rm7f4zf";
+  };
+
+  buildInputs = [
+    cairo
+    libjpeg
+    meson
+    ninja
+    pkgconfig
```
Comment (symphorien) [inline pkgs/tools/graphics/grim/default.nix]: meson, ninja and pkg-config should be nativeBuildInputs

Comment (buffet) [reply pkgs/tools/graphics/grim/default.nix]: Done!

**Comment by dywedir** [comment]
cc @colemickens

**Comment by dywedir** [comment]
@GrahamcOfBorg build grim

**Review by colemickens — APPROVED** [review]

**Pull request merged**